src/annex/annex-create.cpp

```diff
@@ -389,7 +397,7 @@ createOneAnnex( ClassAd * command, Stream * replyStream, ClassAd * reply ) {
 		command->LookupInteger( "EndOfLease", endOfLease );
 
 		if( (! odr->validateAndStore( command, validationError )) || (! validateLease( endOfLease, validationError )) ) {
-			delete cc;
+			// delete cc;
```

Comment:
Can this be removed, or are we leaking?

Reply:
... this is the variable whose constructor was commented out.